test_issue_transaction.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+su - iroha -c '/usr/local/iroha/bin/issue_transaction add peer 172.18.0.3 yUAFTYNh6qkhBMiNgNdg45Wk/XHOGYanuIroY9skfPQ= 1.0 true'
```

Comment:
too long line
bash is able to work with multi-line commands, e.g instead:
`command arg1 arg2 arg1 arg2 arg1 arg2 arg1 arg2 arg1 arg2 arg1 arg2`
you can write smth like:
```
command arg1 arg2\
        arg1 arg2\
        arg1 arg2\
        arg1 arg2\
        arg1 arg2\
        arg1 arg2
```